schedview/collect/efd.py

```diff
@@ -192,3 +192,100 @@ def sync_query_efd_topic_for_night(*args, **kwargs):
     """
     result = _run_async(query_efd_topic_for_night(*args, **kwargs))
     return result
+
+
+def sync_query_latest_in_efd_topic(*args, **kwargs):
+    """Just like query_latest_in_efd_topic, but run in a separate thread
+    and block for results, so it can be run within a separate event loop.
+    """
+    result = _run_async(query_latest_in_efd_topic(*args, **kwargs))
```

Comment:
I wonder if pulling the "time_cut" kwarg out separately, and ensuring that instead of just an astropy time, it is the astropytime.isot that is passed in would be helpful. 
Otherwise, there is no mention in the previous references to time_cut that it has to be either "format=isot" instead of the Time object (as in if x=Time()  ... setting the x.format('isot') to the Time object itself) or that it should be the isot string format passed along (pass in x.isot). )

Reply:
I'd be happy with either -- document taht the time_cut has to be isot or when you call the query_latest_in_efd_typic, make sure that the time_cut is time_cut = time_cut.isot).